requirements.apt

```diff
@@ -0,0 +1,9 @@
+libignition-cmake2-dev
+libignition-common4-dev
+libignition-math6-dev
+libignition-msgs8-dev
+libignition-rendering6-dev
+libignition-tools-dev
+libignition-transport11-dev
+libsdformat12-dev
+xvfb
```

Comment:
maybe we need some other libraries since CICD builds have these errors:
`-- Could NOT find OpenGL (missing: OPENGL_opengl_LIBRARY OPENGL_glx_LIBRARY OPENGL_INCLUDE_DIR) 
-- Searching for <ignition-rendering6> component [ogre2]
-- Looking for ignition-rendering6-ogre2 -- found version 6.6.2
-- Searching for dependencies of ignition-rendering6-ogre2
-- Could NOT find OpenGL (missing: OPENGL_opengl_LIBRARY OPENGL_glx_LIBRARY OPENGL_INCLUDE_DIR) 
-- Looking for ignition-rendering6 - found`